tools/run_tests/sanity/core_banned_functions.py

```diff
@@ -61,3 +65,4 @@
                 errors += 1
 
 assert errors == 0
+assert num_files > 300  # we definitely have more than 300 files
```

Comment:
we probably do but why do we need to check that? :)

Reply:
but maybe just add a comment for that so that no one removes it in the future

Reply:
Good call, done